### ISSUE TITLE
[icon library] Change action for AI Label icon

### ISSUE THREAD

**Issue by alisonjoseph** [carbon-design-system/design-language-website#1374]
This is a duplicate of work that was just completed on the Carbon website 
https://github.com/carbon-design-system/carbon-website/issues/4323

This PR needs to be duplicated here (will also need to update to latest version of Carbon icons, this should be done every 2 weeks in order for the latest icons and pictograms to consistently get pulled in)
https://github.com/carbon-design-system/carbon-website/pull/4336

![Image](https://github.com/user-attachments/assets/76ba78db-3e8f-4359-ba8f-9e13a04a2c0d)

**Comment by leonardo-costa-IBM** [comment]
Hey @alisonjoseph 

The requested update have been made to the `AI Label` icon. You can check it on the [preview](https://cos-static-hosting-brand-center.s3-web.us-south.cloud-object-storage.appdomain.cloud/1493-merge/design/language/iconography/ui-icons/library) url. 
Please, let us know if there is any issue.

FYI: @Lingaraj-13 @andreeavintila

**Comment by leonardo-costa-IBM** [comment]
Hey @andreeavintila 

Yup, we just updated the action related to the AI Label icon only.  We've replaced the "Download" action with a "Launch" action to view the component in Storybook. Should we not consider this update?

**Comment by andreeavintila** [comment]
I don’t think we should do this. IDL serves a different purpose than Carbon, and we should continue to allow SVG downloads. With the latest iconography update, we already provide a direct link to the Carbon library for developers who need access to the code. If we want to clearly signal that this is a component, we can treat this icon differently from the rest of the library and include both the launch and download actions on the card.

**Comment by leonardo-costa-IBM** [comment]
Hey @andreeavintila @alisonjoseph 

Thanks for checking all the context related to this issue and also thanks for your guidance during the update process. As requested, the action related to the AI Label UI icon have been updated from Download to Launch. Also the Launch action is linked to the AI Label component.

This update has been made live. Feel free to check it out.
https://www.ibm.com/design/language/iconography/ui-icons/library

And let us know if there is any other issue,
Thanks in advance.
